packages/apps/tools/src/components/Global/AccountNameField/index.tsx

```diff
@@ -1,68 +1,43 @@
 import { useWalletConnectClient } from '@/context/connect-wallet-context';
-import type { IFormFieldWrapperProps, IInputProps } from '@kadena/react-ui';
-import { FormFieldWrapper, Input } from '@kadena/react-ui';
+import type { ITextFieldProps } from '@kadena/react-ui';
+import { TextField } from '@kadena/react-ui';
 import useTranslation from 'next-translate/useTranslation';
-import type { FC } from 'react';
-import React from 'react';
-import type { FieldError } from 'react-hook-form';
+import { ForwardedRef, forwardRef } from 'react';
+import { FieldError } from 'react-hook-form';
 import * as z from 'zod';
 import { accountInputWrapperStyle } from './styles.css';
 
-interface IAccountNameFieldProps
-  extends Partial<Omit<IFormFieldWrapperProps, 'children'>> {
-  inputProps: Partial<
-    Pick<
-      IInputProps,
-      'id' | 'placeholder' | 'ref' | 'name' | 'onChange' | 'onBlur'
-    >
-  >;
-  error?: FieldError;
-}
-
 export const NAME_VALIDATION = z.string().trim().min(3).max(256);
 
-export const AccountNameField: FC<IAccountNameFieldProps> = ({
-  inputProps,
-  status,
-  helperText,
-  error,
-  ...rest
-}) => {
-  const elementId = 'kd-select-account-input';
-  const mode = 'input';
+interface IAccountNameFieldProps
+  extends Omit<
+    ITextFieldProps,
+    'errorMessage' | 'validationBehavior' | 'onValueChange'
+  > {
+  errorMessage?: string;
+}
+function BaseAccountNameField(
+  props: IAccountNameFieldProps,
+  forwardedRef: ForwardedRef<HTMLInputElement>,
+) {
   const { t } = useTranslation('common');
-  const { onChange, ...restInputProps } = inputProps;
-
   const { selectedAccount, setSelectedAccount } = useWalletConnectClient();
 
-  const lookup = {
-    input: (
-      <Input
-        {...restInputProps}
+  return (
+    <div className={accountInputWrapperStyle}>
+      <TextField
+        validationBehavior="aria"
```

Comment:
should we make validationBehavior="aria" by default?

Reply:
I it is already the default, just added this here from a habit :D